dvclive/live.py

```diff
@@ -180,6 +188,13 @@ def make_summary(self):
         with open(self.summary_path, "w") as f:
             json.dump(summary_data, f, indent=4)
 
+    def make_report(self):
+        if self._report == "html":
+            html_report(self.dir, self.summary_path, self.html_path)
+        if self._auto_open:
```

Comment:
We should probably do it only in case of existing `self._report`. For example:
```
python -c "from dvclive import Live;logger=Live('ble',report=None);logger.log('value', 10);logger.next_step()"
```
will raise some errors, which I wouldn't expect, since I specified that I don't want  a report.